NEWS.md

```diff
@@ -126,6 +126,13 @@
   are printed as a flat expression with the environment printed below.
   Previously they would appear as formulas.
 
+* `UQ()` and `UQS()` are soft-deprecated in order to make the syntax
+  of quasiquotation more consistent. The prefix forms are now ``
+  `!!`() `` and `` `!!!`() `` which is consistent with other R
+  operators (e.g. `` `+`(a, b) `` is the prefix form of `a + b`).
+  These operators are scheduled for deprecation with a warning in
+  rlang 0.3.0.
```

Comment:
That's possibly too soon.

Reply:
Should we give a timeframe instead? E.g. 6 months or 12?